packages/cadl-ranch-specs/http/lro/lro-basic/main.cadl

```diff
@@ -0,0 +1,27 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/cadl-azure-core";
+
+using Cadl.Http;
+using Azure.Core;
+
+@scenarioService("/lro/basic")
+@doc("Basic test for lro")
+namespace Lro.Basic;
```

Comment:
@iscai-msft this being an Azure.Core functionaltiy do you think should we scope those scenarios under azure?

Reply:
I'm not actually sure. For `azure` are we referring to management plane or just azure specific tests. I think its the latter, in which case that makes sense to me

Reply:
yeah AZure.core is technically the base for all azure(ARM is yet another layer on top of it)